src/neuroconv/datainterfaces/behavior/fictrac/fictracdatainterface.py

```diff
@@ -102,21 +98,32 @@ def add_to_nwbfile(
 
         import pandas as pd
 
-        fictrac_data_df = pd.read_csv(self.file_path, sep=",", header=None, names=self.data_columns)
+        # The first row only contains the session start time and invalid data
+        fictrac_data_df = pd.read_csv(self.file_path, sep=",", skiprows=1, header=None, names=self.data_columns)
 
         # Get the timestamps
         timestamps_milliseconds = fictrac_data_df["timestamp"].values
         timestamps = timestamps_milliseconds / 1000.0
-        rate = calculate_regular_series_rate(series=timestamps)  # Returns None if it is not regular
-        write_timestamps = True
-        if rate:
-            write_timestamps = False
 
-        processing_module = get_module(nwbfile=nwbfile, name="Behavior")
+        # Shift to the session start time
+        metadata = metadata or self.get_metadata()
+        interface_session_start_time = extract_session_start_time(self.file_path)
+        conversion_session_start_time = metadata["NWBFile"]["session_start_time"]
+        starting_time = interface_session_start_time.timestamp() - conversion_session_start_time.timestamp()
+        timestamps_shifted = timestamps + starting_time
```

Comment:
How is this supposed to interact with the alignment methods? I'd prefer to use those pathways, and for the `add_to_nwbfile` section to rely on retrieving whatever the user has modified with respect to that

(_e.g._, someone preparsed timestamps separate from FicTrac format and wants to set them explicitly; ditto for manually calculated start time distinct from w/e the header says)

In general we've also never encouraged relying on system times as cached by headers for calculating exact start times

Problem: what if SpikeGLX header (from a separate device) has a clock that is different from that of the FicTrac system (on yet another difference device); comparing the system timestamps is not trustworthy at all in this case, and even if the user wanted to do it they would be encouraged to do so via custom code outside of the interfaces (at the custom converter level or conversion script) to verify that they are OK with that approach - I don't think we should do it automatically in this case

Reply:
This interface is not a `TemporalInterface` so those methods do not exists yet. That said, I think as it is, this implementation will clash with those methods when we upgrade this interface to be temporal. We need to think on this interaction better. I am removing this from this PR and we can keep this discussion here:

https://github.com/catalystneuro/neuroconv/issues/597

Reply:
Removed. I also removed the whole zoneinfo thing. Turns out the utc can be imported from the standard library already so at least in this case no need to do extra fancy thing.